mypy_primer/projects.py

```diff
@@ -1415,7 +1415,7 @@ def get_projects() -> list[Project]:
             mypy_cmd="{mypy} {paths}",
             pyright_cmd="{pyright} {paths}",
             paths=["pwndbg"],
-            deps=["types-gdb"],
+            deps=["setuptools>=71.1", "types-gdb"],
```

Comment:
`pkg_resources` import removed in https://github.com/pwndbg/pwndbg/pull/1983
```suggestion
            deps=["types-gdb"],
```